app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/AllSessionsFragment.kt

```diff
@@ -113,6 +118,28 @@ class AllSessionsFragment :
         binding.sessionsRecycler.smoothScrollToPosition(0)
     }
 
+    override fun saveCurrentSession() {
+        val layoutManager = binding.sessionsRecycler.layoutManager as LinearLayoutManager
+        val scrollState = layoutManager.getScrollState()
+        Prefs.previousSessionScrollPosition = scrollState.anchorPosition
+        Prefs.previousSessionScrollOffset = scrollState.anchorOffset
+    }
+
+    override fun restorePreviousSession() {
+        sessionsViewModel.reopenPreviousSession.observe(this, {
+            if (it != true) return@observe
+            scrollToPreviousSession()
+        })
+    }
+
+    private fun scrollToPreviousSession() {
+        val layoutManager = binding.sessionsRecycler.layoutManager as LinearLayoutManager
+        layoutManager.restoreScrollState(
```

Comment:
Can you make restoreScrollState(scrollState: ScrollState) ? 👀

Reply:
Thanks,  I can make it.